CONTRIBUTING.md

```diff
@@ -76,7 +76,7 @@ for the translators.
 
 When adding user-facing strings to your work, follow these guidelines:
 * Use full sentences. Strings built up out of concatenated bits are hard to translate.
-* Use string formatting instead of interpolation.
+* Use string formatting instead of interpolation. Use the hash format, to give good names to the placeholder values that can be used by translators understand the meaning of the formatted values.
```

Comment:
Typo? `can be used by translators understand` `->` `can be used by translators to understand`